packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js

```diff
@@ -363,6 +367,43 @@ describe('CheckoutModule', () => {
             });
         });
 
+        describe('placeOrder ::', () => {
+            it('should post the order details to the backend.', async () => {
+                // Arrange
+                payload.url = 'http://localhost/opapi/placeorder';
```

Comment:
It looks like this object is mutated in a few places in this test module, do you think it's worth adding a line to a `beforeEach` to reset it for each test?

Reply:
I've updated this now to reset `payload` within the top `beforeEach`. Let me know if you're ok with that, @DamianMullins